.github/workflows/build-test-cxx.yml

```diff
@@ -19,6 +19,12 @@ on:
         required: false
         type: string
         default: 'v5.3.0'
+      itk-module-deps:
+        # Colon-delimited list of ITK remote module dependencies to build.
```

Comment:
Does the other PR add description here? This text is better as a description than a comment.

Reply:
Yes, #30 does that.

Reply:
Agreed, will rebase on #30 and add a description instead